babel.config.js

```diff
@@ -1,53 +1,63 @@
+/**
+ * Babel
+ *
+ * @see https://gist.github.com/ncochard/6cce17272a069fdb4ac92569d85508f4
+ */
+
 module.exports = {
-  presets: ['@babel/react'],
+  presets: ['@babel/preset-env', '@babel/preset-react'],
   plugins: [
     '@babel/plugin-transform-runtime',
     '@babel/plugin-proposal-class-properties',
-    [
-      'styled-components',
-      {
-        displayName: true,
-        ssr: true
-      }
-    ],
```

Comment:
@iwarner Why remove SC? Isn't it better to keep it (does removing it makes any difference?)?